client/lib/analytics/ad-tracking.js

```diff
@@ -505,6 +505,73 @@ function only_retarget() {
 	}
 }
 
+/**
+ * Returns a boolean telling whether we may track the current user.
+ *
+ * @return {Boolean}        Whether we may track the current user
+ */
+export function mayWeTrackCurrentUser() {
```

Comment:
perhaps we should rename the `mayWeTrackCurrentUser()` with something that mentions gdpr, maybe `isGdrpTrackingAllowed()` or something similar?